ctpolicy/ctconfig/ctconfig.go

```diff
@@ -99,15 +99,15 @@ type CTConfig struct {
 	// schema: https://www.gstatic.com/ct/log_list/v3/log_list_schema.json
 	LogListFile string `validate:"required,endswith=.json"`
 	// SCTLogs is a list of CT log names to submit precerts to in order to get SCTs.
-	SCTLogs []string `validate:"gt=0"`
+	SCTLogs []string `validate:"min=1,dive,required"`
 	// InfoLogs is a list of CT log names to submit precerts to on a best-effort
 	// basis. Logs are included here for the sake of wider distribution of our
 	// precerts, and to exercise logs that in the qualification process.
 	InfoLogs []string
 	// FinalLogs is a list of CT log names to submit final certificates to.
 	// This may include duplicates from the lists above, to submit both precerts
 	// and final certs to the same log.
-	FinalLogs []string `validate:"gt=0"`
+	FinalLogs []string `validate:"min=1,dive,required"`
```

Comment:
This should not have `min=1`. It's a totally valid config to have no FinalLogs configured, and it's likely at some future point we might want to do that to reduce burden on logs during times of high load. There's no compliance requirement to submit final certificates; it's something we do because it's handy for the ecosystem to be able to see them (and collect SCTs from them for auditing).